src/utils/deal21Cards.js

```diff
@@ -0,0 +1,9 @@
+const FIRST_CARD = 0;
+const DEAL_NUMBER = 21;
+
+function deal21Cards(deck) {
+  deck.slice(FIRST_CARD, DEAL_NUMBER);
+  return deck;
```

Comment:
Daria pra retornar direto aqui:
```suggestion
  return deck.slice(FIRST_CARD, DEAL_NUMBER);
```

Reply:
Ok! :fist_oncoming: